child_compassion/mappings/childpool_search_mapping.py

```diff
@@ -74,8 +74,8 @@ class AdvancedSearchMapping(OnrampMapping):
         'FieldOffice': 'field_office_ids',
         'Gender': 'gender',
         'HoldingGlobalPartner': 'holding_gp_ids',
-        'ICPID': 'icp_ids',
-        'ICPName': 'icp_name',
+        'FCPID': 'fcp_ids',
+        'FCPName': 'fcp_name',
```

Comment:
they keys of the mappings should always stay as before (it's US message values that didn't change).